pkgs/tools/backup/borgmatic/default.nix

```
@@ -38,7 +39,7 @@ python3Packages.buildPythonApplication rec {
     ruamel-yaml
     requests
     setuptools
-  ];
+  ] ++ lib.optional withApprise apprise;
```

See above remark about optional dependencies.
```suggestion
  ];
```